.ci_support/linux_c_compilertoolchain_ccxx_compilertoolchain_cxxpython2.7.yaml

```diff
@@ -15,7 +15,7 @@ docker_image:
 expat:
 - '2.2'
 glib:
-- '2.56'
+- '2.58'
 gst_plugins_base:
 - '1.12'
```

Comment:
https://github.com/conda-forge/gstreamer-feedstock/pull/17/files#diff-e178b687b10a71a3348107ae3154e44cR1
should be bumped to 1.14

Reply:
Easy enough to fix.

ref: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/651f5b3dad371e92ba1515861d70e2b67c519447/recipe/conda_build_config.yaml#L318-L323

Reply:
...or perhaps these can be dropped from `run` and added to `host` only (followed by a re-render) so that `run_exports` is used instead.